cloud/api/protobuf/kurtosis_backend_server_api.proto

```diff
@@ -28,7 +28,8 @@ service KurtosisCloudBackendServer {
   };
   rpc UnlockPort(UnlockPortRequest) returns(google.protobuf.Empty){};
   rpc LockPort(LockPortRequest) returns(google.protobuf.Empty){};
-  rpc GetUnlockedPorts(GetUnlockedPortsRequest) returns(GetUnlockedPortsResponse){}
```

Comment:
i realize that simply removing this will break for people who might be on the current version of the engine

Reply:
I will keep the previous endpoint;